src/main/resources/assets/javascripts/stores/RunStore.js

```diff
@@ -97,6 +98,9 @@ class RunStore {
   }
 
   onMessage(data) {
+    if (data.state == 'FINISHED' && data.output.location) {
```

Comment:
Should we move the `FINISHED` magic string to a constant file? I forget if we did that in this project :grimacing: